smtr/br_rj_riodejaneiro_viagens/aux_registros_status_viagem.sql

```diff
@@ -0,0 +1,46 @@
+with gps as (
+    SELECT 
+        g.* except(longitude, latitude),
+        ST_GEOGPOINT(longitude, latitude) posicao_veiculo_geo,
+        d.data_versao_efetiva_shapes data_versao
+    FROM `rj-smtr.br_rj_riodejaneiro_veiculos.gps_sppo` g
+    JOIN `rj-smtr.br_rj_riodejaneiro_sigmob.data_versao_efetiva` d
+    ON g.data = d.data
+    WHERE g.data between '2022-02-15' and '2022-02-21'
+),
+shapes as (
+    SELECT 
+        *
+    FROM `rj-smtr.br_rj_riodejaneiro_sigmob.shapes_geom`
+    where 
+        data_versao between (select min(data_versao) from gps) and (select max(data_versao) from gps)
+    AND
+        id_modal_smtr in ('22', '20', 'O')
```

Comment:
```suggestion
        id_modal_smtr in ('22', 'O')
```

Não entra BRT aqui ainda